<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CrossViewClass</class>
 <widget class="QMainWindow" name="CrossViewClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1050</width>
    <height>661</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CrossView</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTableView" name="CrossTableView">
       <attribute name="horizontalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>16</number>
       </attribute>
       <attribute name="horizontalHeaderMinimumSectionSize">
        <number>1</number>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderDefaultSectionSize">
        <number>16</number>
       </attribute>
       <attribute name="verticalHeaderMinimumSectionSize">
        <number>1</number>
       </attribute>
      </widget>
      <widget class="QTableView" name="PaletteTableView">
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>50</number>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1050</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoadFile"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionShowPalette"/>
    <addaction name="separator"/>
    <addaction name="actionZoomIn"/>
    <addaction name="actionZoomOut"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTools"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionLoadFile">
   <property name="icon">
    <iconset resource="CrossView.qrc">
     <normaloff>:/CrossView/Resources/pic/open.png</normaloff>:/CrossView/Resources/pic/open.png</iconset>
   </property>
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="actionShowPalette">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="CrossView.qrc">
     <normaloff>:/CrossView/Resources/pic/palette.png</normaloff>:/CrossView/Resources/pic/palette.png</iconset>
   </property>
   <property name="text">
    <string>Show Palette</string>
   </property>
  </action>
  <action name="actionZoomIn">
   <property name="icon">
    <iconset resource="CrossView.qrc">
     <normaloff>:/CrossView/Resources/pic/zoomin.png</normaloff>:/CrossView/Resources/pic/zoomin.png</iconset>
   </property>
   <property name="text">
    <string>Zoom In</string>
   </property>
  </action>
  <action name="actionZoomOut">
   <property name="icon">
    <iconset resource="CrossView.qrc">
     <normaloff>:/CrossView/Resources/pic/zoomout.png</normaloff>:/CrossView/Resources/pic/zoomout.png</iconset>
   </property>
   <property name="text">
    <string>Zoom Out</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="CrossView.qrc"/>
 </resources>
 <connections/>
</ui>
